extension/src/background/websocket-client.ts

```diff
@@ -20,10 +20,46 @@ export default class WebSocketClient {
                 );
                 resolve();
             };
+            this.webSocket.onmessage = (event: any) => {
```

Comment:
The type is string here no?

Reply:
Does not seem to be so simple https://stackoverflow.com/questions/9462879/how-do-i-tell-the-type-of-websocket-onmessages-parameter